src/Stdin.vue

```diff
@@ -103,7 +108,7 @@ export default {
   methods: {
     handle () {
       if (this.isInProgress) return
-
+      this.locPrompt = this.prompt.substring()
```

Comment:
I don't understand why `substring` is necessary. Would you explain it?

Reply:
It is not necessary, i just removed it.
Was thinking that a copy of `this.prompt` is required, because it is reactive, but it is not necessary.